dfx-library/iirfilter.cpp

```diff
@@ -179,7 +181,13 @@ void dfx::IIRFilter::setCoefficients(Coefficients const& inCoefficients)
 dfx::IIRFilter::Coefficients const& dfx::IIRFilter::setCoefficients(FilterType inFilterType, double inFrequency, double inQ, double inGain)
 {
 	assert(inFrequency > 0.);
+	assert(!std::isinf(inFrequency));
```

Comment:
BTW you can use std::isfinite() to check both at the same time, and it also puts the assertion in positive form.

Reply:
ooo I hadn't met isfinite... :)

Reply:
...although actually I remember now that I like them split in asserts because then I can see which specific type of gremlin occurred, which can be a helpful distinction for debugging. Though I noted another spot where I was needlessly doing isnan together with isfinite (so I guess I had met it, but didn't realize it covered NaNs).